lms/static/sass/course/courseware/_courseware.scss

```diff
@@ -102,6 +102,13 @@ div.course-wrapper {
     @extend .content;
     padding: ($baseline*2);
     line-height: 1.6;
+
+    .xblock {
```

Comment:
Looks good, if we want to use this elsewhere, it'd make sense to create a Sass extend, but we can follow up and do that after this fix gets in.